src/BlazorWebView/tests/MauiDeviceTests/MauiBlazorWebView.DeviceTests.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
```

Comment:
I made the test project be a proper "MAUI Blazor Hybrid" app so that static web assets would get pulled in. Previously no tests needed this, but in this PR I added tests that rely on JS Interop through a scoped service, and that uses a static JS file in an RCL. Without this CSPROJ change the static JS files wouldn't get pulled in.